draco/analysis/transform.py

```diff
@@ -217,9 +224,9 @@ def pack_product_array(arr):
         # Construct the equivalent stack reverse_map for the telescope instance.  Note
         # that we identify invalid products here using an index that is the size of the stack axis.
         feedmask = pack_product_array(self.telescope.feedmask)
-        bt_rev = np.array(zip(
+        bt_rev = np.fromiter(zip(
                  np.where(feedmask, pack_product_array(self.telescope.feedmap), self.telescope.npairs),
-                 np.where(feedmask, pack_product_array(self.telescope.feedconj), 0)),
+                 np.where(feedmask, pack_product_array(self.telescope.feedconj), 0))),
```

Comment:
You need to also revert the added `)` here:
```suggestion
                 np.where(feedmask, pack_product_array(self.telescope.feedconj), 0)),
```